ansible/playbooks/install-node-deps.yml

```diff
@@ -1,5 +1,7 @@
 ---
 - hosts: redpanda
+  vars: 
+    node_exporter_version: 1.5.0
```

Comment:
Can we make this into `node_exporter_version: node_exporter_version | default('1.5.0')` please? This allows the user to override it if they really need to.